### PR TITLE
Fix/sniff ecms se collaboration

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "platform/src/test/resources/data"]
 	path = platform/src/test/resources/data
 	url = https://github.com/exoplatform/platform-qa-ui-data.git
-	branch = feature/external-data-set
+	branch = data-for-ecms
```
Comment (thomasdelhomenie) [inline .gitmodules]: Change to discard

File: ecms/src/main/java/org/exoplatform/platform/qa/ui/ecms/pageobject/SiteExplorerHome.java [hunk 2]
```diff
@@ -942,19 +937,22 @@ public void addEditComment(String content, boolean isAdd) {
...
+    refresh();
     info("Input a content to the frame");
-    evt.inputDataToCKEditor(ELEMENT_FILEFORM_BLANK_CONTENT, content);
+    $(byId("comment")).sendKeys(content);
+/*    $(ELEMENT_FILEFORM_BLANK_CONTENT).click();
+    switchTo().frame($(ELEMENT_FILEFORM_BLANK_CONTENT));
+    $(byXpath("/html/body")).sendKeys(content);
     info("Switch to parent window");
-    evt.switchToParentWindow();
+    switchTo().defaultContent();*/
```
Comment (thomasdelhomenie) [inline ecms/src/main/java/org/exoplatform/platform/qa/ui/ecms/pageobject/SiteExplorerHome.java]: Commented code to remove

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+import org.exoplatform.platform.qa.ui.core.PLFData;
+import org.exoplatform.platform.qa.ui.core.context.BugInPLF;
+import org.exoplatform.platform.qa.ui.ecms.pageobject.CreateNewDocument;
+import org.exoplatform.platform.qa.ui.ecms.pageobject.SiteExplorerHome;
+import org.exoplatform.platform.qa.ui.selenium.ManageAlert;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.selenium.platform.NavigationToolbar;
+
+/**
+ * @date 15-January-2015
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java]: To remove

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition: A tag is already created.</li> Step Number: 1 Step Name:
+   * Step 1: Add a tag Step Description: - Select a document or uploaded file -
+   * Click on Tag button in the action bar (if not, select Tag from More drop
+   * -down menu) - Put name for a tag - Click Add button, the Close button Input
+   * Data: Expected Outcome: Node is added tag. You can find document using tag in
+   * Tag cloud of FE
+   */
+  @Test
+  @BugInPLF("ECMS-7676")
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java]: This issue has been fixed.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+    $(byId("UITagExplorer")).find(byText(content2)).should(Condition.visible);
+    info("Test 06: Edit a tag");
+    siteExplorerHome.editTag(content2, content3);
+    info("Test 07: Delete a tag");
+    siteExplorerHome.deleteTag(content3);
+    siteExplorerHome.deleteData(title);
+  }
+
+  @Test
+  @BugInPLF("ECMS-7676")
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java]: This issue has been fixed.

File: .gitmodules (hunk 1 above)
Comment (ilyeskallala) [reply .gitmodules]: done

File: ecms/src/main/java/org/exoplatform/platform/qa/ui/ecms/pageobject/SiteExplorerHome.java (hunk 2 above)
Comment (ilyeskallala) [reply ecms/src/main/java/org/exoplatform/platform/qa/ui/ecms/pageobject/SiteExplorerHome.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java (hunk 3 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java (hunk 4 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java]: tag removed

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java (hunk 5 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsSECollaborationTestIT.java]: tag removed

**Pull request merged**